.circleci/config.yml

```diff
@@ -1,11 +1,42 @@
-version: 2
+version: 2.1
+
+commands:
+    configure-composer-auth:
+        description: Configure composer auth
+        steps:
+            - run: mkdir -p ~/.composer/cache
+            - run:
+                name: Add GitHub token for composer
+                command: echo ${COMPOSER_AUTH} > ~/.composer/auth.json
```

Comment:
why?

Reply:
I chose to write the token in a file, as we are sometimes running Composer through a shell script, and it seems the environment variable doesn't reach it…